src/content/docs/en/guides/markdown-editor/stackeditpro.mdx

```diff
@@ -0,0 +1,79 @@
+:::
+
+### Adding new Markdown files to your Astro repo from StackEditPro
+
+To add a new document to a linked Astro repo hosted on GitHub, first create a new note in StackEditPro then publish it to a file location within your repository. 
+
+<Steps>
+1. From your HackMD dashboard, create a new note.
+2. Add page content to your note in Markdown, including any frontmatter YAML.
+3. From the StackEdit logo menu, select `Synchronize > Save on GitHub`.
+4. Provide the repository, filepath, and name of the new file, including file extension (`.md` or `.mdx`), as you would like it to appear in your Astro repo. You can optionally specify a branch.
+5. When you have provided all the necessary information, select `OK` to commit the new file to your Astro repository.
+6. To confirm that your changes have been successfully applied, you can visit your repository on GitHub.com and verify that the file content has been updated.
+</Steps>
+
+### Editing an existing file from your Astro repository in StackEditPro
+
+You can edit any existing file from a linked GitHub repository by opening from GitHub. A new linked StackEditPro note will be created in your dashboard.
+
+<Steps>
+1. From the StackEdit logo menu, select `Synchronize > Open from GitHub`.
+2. Provide the repository, filepath and name of the file, including file extension (`.md` or `.mdx`), as it exists in your Astro repo. You can optionally specify a branch.
+3. StackEditPro will create a new local note in your dashboard using the name of the file as the title.
+4. Make any desired changes to your content in the StackEditPro Markdown editor, including frontmatter YAML if needed.
+5. When you are ready to publish your changes back to GitHub, `Synchronize` again.
+6. To confirm that your changes have been successfully applied, you can visit your repository on GitHub.com and verify that the file content has been updated.
+</Steps>
+
+### Updating your Astro content from a linked StackEditPro note
+
+You can update an existing StackEditPro note that has been linked to a file in your Astro repo, then push your changes back to your project. This is a convenient way for you or your collaborators to maintain and update the content of files that change regularly.
+
+:::caution
+When a StackEditPro note has been linked with a GitHub file, your local note is not automatically in sync with the GitHub version. Therefore, it is important to first `Syncronize` any changes to be sure you are working with an updated copy of your content.
+
+If you `Syncronize` with local changes without pulling first, your file on GitHub (the "server" copy) will overwrite the contents of your local note to prevent data loss from your GitHub repository. A merge conflict report will be generated as a new StackEdit note in your dashboard with instructions for further options for data recovery.
```

Comment:
```suggestion
If you `Synchronize` with local changes without pulling first, your file on GitHub (the "server" copy) will overwrite the contents of your local note to prevent data loss from your GitHub repository. A merge conflict report will be generated as a new StackEdit note in your dashboard with instructions for further options for data recovery.
```